### PR TITLE
Feat/multi language

### PR DESCRIPTION
**Pull request by toncatalansuazo**
it, es, en added. Using next-translate

### CODE REVIEW

File: i18n.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "locales": ["en", "es", "it"],
+  "defaultLocale": "en",
+  "pages": {
+    "*": ["common"]
+  }
+}
```
Comment (EgidioCaprino) [inline i18n.json:7]: We should add an empty line at the end of each file.

File: locales/en/common.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "intro-title": "Technology that makes your life easier",
+  "button-continue": "continue",
+  "button-contact-us": "contact us",
+  "WelcomeTo React": "Bienvenue à React et react-i18next",
```
Comment (EgidioCaprino) [inline locales/en/common.json:5]: Is this French? Is that correct? In appears in the other files too.

**Review by EgidioCaprino — CHANGES_REQUESTED** [review]
Looks good, just few minor changes. Also, I think we should have significant name for image files, cause they get indexed by search engines.

File: locales/en/common.json (hunk 2 above)
Comment (toncatalansuazo) [reply locales/en/common.json:5]: I have to correct that

Comment (toncatalansuazo) [reply locales/en/common.json]: fixed in commit https://github.com/capcatit/capcatit.github.io/pull/14/commits/0c434f527307558c2056b8e649f286d23177144d

File: i18n.json (hunk 1 above)
Comment (toncatalansuazo) [reply i18n.json:7]: done in commit https://github.com/capcatit/capcatit.github.io/pull/14/commits/0c434f527307558c2056b8e649f286d23177144d

**Review by EgidioCaprino — APPROVED** [review]

**Pull request merged**